### ISSUE TITLE
GSE178360 - LungMapVisium

### ISSUE THREAD

**Issue by arschat** [ebi-ait/hca-ebi-wrangler-central#1123]
### Associated files

* Google Drive: [folder](https://drive.google.com/drive/folders/1gAt85j5rsd3fJBgAACuYylBEthCS1ZnW?usp=sharing)
* ingest: [e5fe8274-3769-4d7d-aa35-6d33c226ab43](https://contribute.data.humancellatlas.org/projects/detail?uuid=e5fe8274-3769-4d7d-aa35-6d33c226ab43&tab=project)

### Published study links

* Paper: [Human distal lung maps and lineage hierarchies reveal a bipotent progenitor.](https://www.nature.com/articles/s41586-022-04541-3)

* Accessioned data:
[GSE178360 scRNA-seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178360)
[GSE178519 organoids](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178519)
[GSE178361 visium](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178361)

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [ ] Are the analysis files suitable for CellxGene? If something is missing get in touch with the authors to request it
- [x] Upload sheet to validate metadata
- [x] Transfer raw files to ingest to validate data files
- [ ] Check linking using ingest graph validator
- [ ] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Submit dataset to Production 
- [ ] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by arschat** [comment]
There is metadata table, but there is no mention about the Visium donor.
Emailed contributors about it, and corresponding author replied redirecting us to first author.
Waiting for reply with clarifications from first author.

**Comment by ofanobilbao** [comment]
It contains 3 datasets, scRNA-seq, Visium and Organoids. Might be ready by end of today or Monday for secondary review. Still some confirmation from authors required

**Comment by arschat** [comment]
Authors verified:
> Donor 1	DD046Q	scRNA-seq
> Donor 2	DD047Q	scRNA-seq & AT2 cultures
> Donor 3	DD073R	scRNA-seq & Visium

and that all specimens were collected _post mortem_

Notes & questions:
- Visium sequence & analysis files are all derived from the imaged_specimen or there should be another cell_suspension?
- I am not sure about the cell_line tab. In paper they mention an intermediate cell culture before the organoids but not specifically a cell line. There is reference to a paper about AT2 isolation though, that includes specific growth media. There is also dissociation & enrichment protocols prior the organoid creation therefore I concluded into including a cell_line tab.
- Control organoid is mentioned as an organoid although it is not described in the _Organoid culture_ method part. In paper they imply that control is undifferentiated AT2 cells, in comparison with differentiated organoids. Not sure if I should skip organoid and reference cell_line as input for CS. I included control organoid entity as a undifferentiated organoid, since the "control organoid" is mentioned in the paper.

metadata validating...

**Comment by arschat** [comment]
* Moved `scalefactor json`s with the image metadata into the `image file` tab as discussed.
* The `filtered_feature_bc_matrix` matrices contain both raw and unnormalized counts but there are less cells compared to the `raw_feature_bc_matrix` matrices. I think it's the 10x filtration of mostly empty cells. 
Therefore, I sticked with the `count matrix` ontology term.

Added all the other changes and exported.
Import form sent.
